README.md

```diff
@@ -104,6 +104,17 @@ To learn more abut what `eksctl` can do check [eksctl.io](https://eksctl.io). A
 
 Code contributions are very welcome. If you are interested in helping make `eksctl` great then see our [contributing guide](CONTRIBUTING.md).
 
+## Releases
+
+`eksctl` will be releasing on a 2 week cadence (minor releases) with extra patch releases in between as needed.
+
+This versioning scheme will be effective from version 0.2.0 onwards.
+
+RC releases will be published for testing purposes 2 working days prior to every minor release. At the end of this testing period, and after any regressions have been found and fixed, the release will take place.
+
+It is highly discouraged to use RC releases in any production environment. They are for testing purposes only.
+
+
```

Comment:
Let's keep it single blank line please...

Reply:
+1